tests/core/switch_session_test.py

```diff
@@ -74,12 +87,25 @@ def test_close_session(self):
         self.session_manager.close_session('patate')
 
         with self.assertRaises(UnknownResource):
-            self.session_manager.get_switch_for_session('patate')
+                self.session_manager.get_switch_for_session('patate')
+
+    @patch('netman.core.switch_sessions.SwitchSessionManager.logger')
+    def test_close_session_logs_if_remote_remove_fails(self, mock_log):
```

Comment:
Only to ensure that exception is caught by asserting that the logging done inside the exception is called.